docs/install.md

```diff
@@ -19,7 +19,7 @@ docker pull quay.io/goswagger/swagger
 #### For Mac And Linux users:
 
 ```bash
-alias swagger='docker run --rm -it  --user $(id -u):$(id -g) -e GOPATH=$HOME/go:/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger'
+alias swagger='docker run --rm -it  --user $(id -u):$(id -g) -e GOPATH=$(which go):/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger'
```

Comment:
I think it would be better to use `$(go env GOPATH)`

Reply:
Perfect!! Should i am raising new PR with this option.

Reply:
Thanks for this suggestion @casualjim . I thought we need `bin` here. Made new commit with suggested changes: https://github.com/go-swagger/go-swagger/commit/ebc939f1d8b31af7c60feeb36bc07a2b40778d86